CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.5.0]
+- Add support for Finshark (Open Banking).
+- Update minimum PHP supported version to 7.4
```

Comment:
Are you sure about this?

Upgrading PHP is not an easy task. Have you verified whether merchants are able to do this change easily?

Reply:
@emicha We have aligned our PHP version with WooCommerce as WooCommerce now requires PHP7.4 as the minimum supported version.
If a merchant has an older PHP version, they'll see a warning when updating the plugin. To do the update, they'll have to upgrade their PHP version. like the below screenshot.

<img width="1229" alt="Screenshot 2023-10-26 at 2 25 28 PM" src="https://github.com/AltaPay/plugin-wordpress/assets/39646587/20acf2bd-c31f-4dc2-8ce6-e988bd674cbd">

Another reason to update the PHP version is to include dependencies with the latest versions as we build the plugin with a minimum supported version for the marketplace.